Analysis of tiny flag (#23)
BZFlag-Dev/bzflag.org: _documentation/user/flags/tiny.md

summary: Tank is small and can get through small openings. Very hard to hit.
---

The Tiny (T) flag allows the player to be smaller in tank size than normal, all while retaining all other properties, such as the tank speed, shot speed, or jump height.

> It is configured by the `_tinyFactor` server setting, which is multiplied by the normal tank size to get shrunken `_tankLength` and `_tankWidth` values for the tank holding the flag.

Reviewer: this is good information, but is it more appropriate in the Server Settings section?

Author: @jwmelto thanks for the feedback; so far, we have only used the Server Settings section to *list* the server settings that directly affect the flag, everything else was explained or expanded in either Summary or Tactics sections. We can change that, but then we will have to reflect it in all other closed cases as well.

Reviewer 2: I think this is something we can definitely revisit once we start officially moving over BZDB variable documentation to the new site.

Since this has started to grow stale, I'll merge this as-is.

> It is declared by the value of the `_tinyFactor` server setting, which is multiplied by normal tank size, to shrink a tank holding the flag. Manipulation of the size is a combination of changing `_tankLength` and `_tankWidth` in parallel.

Reviewer 2: ```suggestion
It is configured by the `_tinyFactor` server setting, which is multiplied by the normal tank size to get shrunken `_tankLength` and `_tankWidth` values for the tank holding the flag.
```

Maybe something like this? Thoughts?

## Tactics

> Tiny can be utilized as a means of easier bullet evasion. Since the tank is smaller than normal, it can slip through bullet spreads that a tank of normal size could not. The closer to zero the `_tinyFactor` is set, the more potent Tiny is.

Reviewer 2: ```suggestion
Tiny can be utilized as a means of easier bullet evasion. Since the tank is smaller than normal, it can slip through bullet spreads that a tank of normal size could not. The closer to zero the `_tinyFactor` is set, the harder it is to hit.
```

> The closer to zero the `_tinyFactor` is set, the more potent Tiny is.

This is opinionated.
